messages/run-dfa.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
 		"outfileDescriptionLong": "Write output to a file.",
 		"projectdirDescription": "root directory of project",
 		"projectdirDescriptionLong": "Root project directory. Must be paths, not globs. Multiple values can be specified as a comma-separated list",
+		"ruledisablewarningviolationDescription": "??????? (default: false). Alternatively, set value using environment variable `SFGE_RULE_DISABLE_WARNING_VIOLATION`",
+		"ruledisablewarningviolationDescriptionLong": "??????. Inherits value from SFGE_RULE_DISABLE_WARNING_VIOLATION env-var if set.",
```

Comment:
I have absolutely no idea how to describe this flag. Suggestions are welcome. Please provide them.
Sidenote: I used the `ignoreparseerror` descriptions as the template for these ones, but while doing so I noticed that the `DescriptionLong` is actually *shorter* than the `Description`. Perhaps we should swap them?

Reply:
Agree that swapping them for `ignoreparseerror` is a good idea.

As for new text, how about something like:
```
description: disable warning violations from Graph Engine
descriptionLong: Disable warning violations such as warnings on StripInaccessible READ access to get only high severity violations.
```